plugins/foreman_discovery/2.0/index.md

```diff
@@ -351,7 +376,17 @@ click on *Refresh features* button and it will appear immediately.
 In *Infrastructure > Subnets* select the desired *Discovery Proxy* for each
 appropriate Subnet.
 
-## 3.3 Permissions
+## 3.3 Hammer CLI Foreman Discovery plugin
+
+The plugin must be configured in ```HAMMER_LOCATION/cli.modules.d/foreman_discovery.yml```, examples of [HAMMER_LOCATION](https://github.com/theforeman/hammer-cli/blob/master/doc/installation.md#locations)
```

Comment:
Just to clear up this sentence:

    The plugin must be enabled in ```cli.modules.d/foreman_discovery.yml``` (see [Hammer config directories](http...)) as follows: